### PR TITLE
Music: fix zero key

### PR DESCRIPTION
**Pull request by breville**
Fixes an issue in which switching to a pack with a `key` of `0` was not applying that key.

### CODE REVIEW

File: apps/src/music/player/MusicLibrary.ts [hunk 1]
```diff
@@ -244,9 +244,9 @@ export default class MusicLibrary {
     }
     const folder = this.getFolderForFolderId(this.currentPackId);
     // Read key from the folder, or the first sound that has a key if not present on the folder.
-    return (
-      folder?.key || folder?.sounds.find(sound => sound.key !== undefined)?.key
-    );
+    return folder?.key !== undefined
+      ? folder?.key
+      : folder?.sounds.find(sound => sound.key !== undefined)?.key;
```
Comment (thomasoniii) [inline apps/src/music/player/MusicLibrary.ts]: nullish coalescing operator for the win!

```
folder?.key ?? folder?.sounds.find(sound => sound.key !== undefined)?.key
```

Lets us keep the simpler logic w/o adding the ternary.

**Review by thomasoniii — CHANGES_REQUESTED** [review]
Made a note on the PR - use the nullish coalescing operator instead of a ternary.

File: apps/src/music/player/MusicLibrary.ts (hunk 1 above)
Comment (breville) [reply apps/src/music/player/MusicLibrary.ts]: Nice, thanks!  Done.

**Review by sanchitmalhotra126 — APPROVED** [review]
Thanks for catching this!

**Review by thomasoniii — APPROVED** [review]
lgtm 🚀

**Pull request merged**